docs/DeveloperGuide.md

```diff
@@ -806,41 +812,162 @@ testers are expected to do more *exploratory* testing.
+
+    2. **Test case:** `updatephoto 1 path/docs/images/johndoe.png`<br>
+       **Expected:** The first `Person` in the list will have profile photo updated to the picture "johndoe.png". Success details shown in the status message.
+
+    3. **Test case:** `updatephoto 1 path/C:/documents/test.docx`<br>
+       **Expected:** The first `Person` in the list will not have profile photo updated. Error details shown in the status message.
+
+2. To see if the `Person`'s profile photo is updated, use the [find command](https://ay2324s1-cs2103t-t13-2.github.io/tp/UserGuide.html#contact-search) to search for the `Person` and verify the profile photo.
+
+### Adding note to a person
+
+1. Adding note to a `Person`:
+
+    1. **Prerequisites:** List all `Persons` in CampusConnect using the `list` command.
+
+    2. **Test case:** `addnote 1 This is a sample note for the person.`<br>
+       **Expected:** The first `Person` in the list will have a new note stating "This is a sample note for the person.". Success details shown in the status message.
+
+    3. **Test case:** `addnote 1 This person is very funny! 😀`<br>
+       **Expected:** The first `Person` in the list will not have a new note stating "This person is very funny! 😀". Error details shown in the status message.
+
+2. To see if the new note is added to `Person`, use the [viewnotes command](https://ay2324s1-cs2103t-t13-2.github.io/tp/UserGuide.html#view-notes-of-contact-viewnotes) and verify the details of the notes.
+
+### Paying money to a person
+
+1. Paying money to a `Person`:
+
+    1. **Prerequisites:** List all `Persons` in CampusConnect using the `list` command.
+
+    2. **Test case:** `pay 1 10`<br>
+       **Expected:** The first `Person` in the list will be paid $10. Success details shown in the status message.
+
+    3. **Test case:** `pay 1 10.555`<br>
+       **Expected:** The first `Person` in the list will not be paid $10.555. Error details shown in the status message.
+
+   4.  **Test case:** `pay 1 50000`<br>
+       **Expected:** The first `Person` in the list will not be paid $50000. Error details shown in the status message.
+
+2. To see if the money is paid to `Person`, use the [find command](https://ay2324s1-cs2103t-t13-2.github.io/tp/UserGuide.html#view-notes-of-contact-viewnotes) to search for the `Person` and verify the money paid.
+
+
+### Owing money to a person
+
+1. Owing money to a `Person`:
+
+    1. **Prerequisites:** List all `Persons` in CampusConnect using the `list` command.
+
+    2. **Test case:** `owe 1 10`<br>
+       **Expected:** The first `Person` in the list owes you $10. Success details shown in the status message.
+
+    3. **Test case:** `owe 1 10.555`<br>
+       **Expected:** The first `Person` in the list will not owe you $10.555. Error details shown in the status message.
+
+   4.  **Test case:** `owe 1 50000`<br>
+       **Expected:** The first `Person` in the list will not owe you $50000. Error details shown in the status message.
```

Comment:
When we use `own` command for a given contact, we ourselves should be the one owing to that person right?